src/main/components/uml-element/resizable/resizable.tsx

```diff
@@ -60,39 +91,94 @@ export const resizable =
         return (
           <WrappedComponent {...props}>
             {props.children}
-            <Handle onPointerDown={this.onPointerDown} />
+            <HandleBottomRight
+              onPointerDown={(e) => {
+                this.onPointerDown(e, 'bottomRight');
+              }}
+            />
+            <HandleTopLeft
+              onPointerDown={(e) => {
+                this.onPointerDown(e, 'topLeft');
+              }}
+            />
+            <HandleTopRight
+              onPointerDown={(e) => {
+                this.onPointerDown(e, 'topRight');
+              }}
+            />
+            <HandleBottomLeft
+              onPointerDown={(e) => {
+                this.onPointerDown(e, 'bottomLeft');
+              }}
+            />
           </WrappedComponent>
         );
       }
 
-      private resize = (width: number, height: number) => {
-        width = Math.round(width / 20) * 20;
-        height = Math.round(height / 20) * 20;
+      private resize = (width: number, height: number, resizeFrom: string) => {
+        width = Math.round(width / 5) * 5;
```

Comment:
I'd think about if we should reset it to 20, now that we found another solution for the repositioning issue.